### ISSUE TITLE
[WIP, false-positive] Magic methods validity: __set_state

### ISSUE THREAD

**Issue by Fleshgrinder** [kalessil/phpinspectionsea#102]
The magic method [`__set_state`](https://php.net/language.oop5.magic#object.set-state) is incorrectly catched by the _magic methods validity_ inspection as being a non-magic method and thus displays an error if that method is encountered.

## MWE
```php
<?php

class VarExportable {

    public static function __set_state(array $state) {}

}
```

**Comment by kalessil** [comment]
Thank you for reporting @Fleshgrinder, will be fixed with the next release.
